pkg/version/version.go

```diff
@@ -49,3 +51,36 @@ func Get() Info {
 		Platform:   fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+// Error is an error for versions in case it is desired to check an error for this type
+type Error struct {
+	Component       string
+	ExpectedVersion string
+	Version         string
+}
+
+// Error is a required function for type Error
+func (e Error) Error() string {
+	return fmt.Sprintf("expected version: %s, found vresion: %s", e.ExpectedVersion, e.Version)
+}
+
+//Valid returns nil if expected version is meet by actual using solely Major.Minor
+func Valid(component string, actual *semver.Version, expected *semver.Version) error {
```

Comment:
so we're ignoring patch here? Should we note/document it somewhere?

Reply:
this is the reason for not using compare... we don't want to use patch as part of the comparison... which also enables the behavior that `1.13` means `1.13.1` and greater.

Reply:
notes /docs?   isn't that what `nil if expected version is meet by actual using solely Major.Minor` basically says?    we could be more explicit if we need to be

Reply:
uh, sorry, I don't understand that sentence :(

Reply:
is it saying that nil is returned if the two differ only in patch version?

Reply:
I'll rephrase it :)